app/script/util/ValidationUtil.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+window.z = window.z || {};
+window.z.util = z.util || {};
+
+z.util.ValidationUtil = {
+  // ToDo: Move z.util.is_valid_username here
+  // ToDo: Move z.util.is_valid_phone_number here
+  // ToDo: Move z.util.is_iso_string here
+  // ToDo: Move z.util.is_valid_email here
+  // ToDo: Move z.util.is_same_location here
+  asset: {
+    legacy: (asset_id, conversation_id) => {
+      if (!z.util.ValidationUtil.isUUID(asset_id) ||
+          !z.util.ValidationUtil.isUUID(conversation_id)) {
+        throw new z.util.ValidationUtilError('Invalid asset_id / conversation_id');
+      }
+    },
+    retentionPolicy: (str) => {
```

Comment:
The concernt that there is no camelCase in webapp code is still valid. Should be `retention_policy`.